src/lib/storage/planned_volumes_list.rb

```diff
@@ -29,7 +29,7 @@ module Storage
     # Set of PlannedVolume elements
     #
     # Implements Enumerable and provides some extra methods to query the set
```

Comment:
It comes from the previous line "Set of PlannedVolume elements". But I can clarify if needed. Using "set" just as a synonym for "list" or "collection" with no particular reason. I will rephrase it a bit.